Silicon/CommonSocPkg/Library/WatchDogTimerLib/WatchDogTimerLib.c

```diff
@@ -23,6 +23,7 @@
 #define B_ACPI_IO_OC_WDT_CTL_LCK                      BIT12
 #define B_ACPI_IO_OC_WDT_CTL_TOV_MASK                 0x3FF
 #define B_ACPI_IO_OC_WDT_CTL_SCRATCHPAD_MASK          0xFF0000
+#define BOOT_COUNT_MASK                               0xC0000
 
```

Comment:
Change BOOT_COUNT_MASK to B_ACPI_IO_OC_WDT_SCRATCHPAD_BOOT_COUNT_MASK to align with other definition?